src/main/scala/is/hail/variant/MatrixTable.scala

```diff
@@ -503,13 +517,11 @@ class MatrixTable(val hc: HailContext, val ast: MatrixIR) extends JoinAnnotator
   def groupSamplesBy(keyExpr: String, aggExpr: String): MatrixTable = {
```

Comment:
Does we want to make keyExpr compound?

Reply:
we totally do, but I want to do that in a separate PR